metapool/plate.py

```diff
@@ -439,6 +440,52 @@ def _autopool_norm(pool_failures, total_nmol, min_conc, sample_concs,
     return plate_df
 
 
+def merge_plate_dfs(plate_df_one, plate_df_two, wells_col=PM_WELL_KEY,
```

Comment:
A couple of comment here:
1. Just for clarity, the main reason for this method is to "clarify" the warning, right? Or is there another reason to reimplement pd.merge?
2. As far as I can tell the only place the default values for the 2 plates (plate_df_one, plate_df_two) is used is in the tests; if that's correct, why set defaults?

Reply:
1) Right, this just *wraps* `pd.merge`, and the reason is that `pd.merge` will NOT give a warning about this situation (since, for arbitrary dataframes, there's nothing wrong with doing an inner merge like this).  However, when we happen to know that the dataframes we are trying to merge in these calls are supposed to represent *plates*, we should be a bit suspicious about a situation where the plates we are merging (by well) don't have the same wells on them :)

2) Fair enough; I have taken out defaults for those and the well column name.